repo/packages/A/artifactory-lb/3/package.json

```diff
@@ -0,0 +1,26 @@
+{
+  "packagingVersion": "3.0",
+  "name": "artifactory-lb",
+  "version": "5.11.1",
```

Comment:
Minor: I would follow a format baseTechVersion-PackageVersion
i.e., 5.11.1-1.0

Reply:
My team likes to keep the existing convention. ie: "5.11.1" ... Thanks for your recommendation.